src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -731,11 +731,15 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="activatePackages" value="Ctrl+7"/>
          <shortcut refid="activateEnvironment" value="Ctrl+8"/>
          <shortcut refid="activateViewer" value="Ctrl+9"/>
-         <shortcut refid="activateVcs" value="Ctrl+F1"/>
-         <shortcut refid="activateBuild" value="Ctrl+F2"/>
-         <shortcut refid="activateConnections" value="Ctrl+F5"/>
-         <shortcut refid="activateFindInFiles" value="Ctrl+F6"/>
-         <shortcut refid="newSourceColumn" value="Ctrl+F7"/>
+         <shortcut refid="activateVcs" value="Meta+F1" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="activateVcs" value="Ctrl+F1" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="activateBuild" value="Meta+F2" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="activateBuild" value="Ctrl+F2" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="activateConnections" value="Ctrl+F5" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
```

Comment:
Should we add a comment here that `Meta+F5` is reserved on mac so someone doesn't add it while trying to be helpful?

Reply:
Was thinking we need a reference like that, there are so many platform-specific shortcut rules and I've yet to find a comprehensive site. Anyway, this one does seem very likely to get put back in given the pattern surrounding it so I will add a comment.